install/tests/Makefile

```diff
@@ -364,9 +380,12 @@ kots-install: version-flag = $(if $(version:-=),--app-version-label=$(version),)
 kots-install: preflight-flag = $(if $(preflights:true=),--skip-preflights,)
 kots-install: license-file = $(if $(license_community_$(channel)),$(license_community_$(channel)),"../licenses/$(channel).yaml")
 kots-install: install-kots-cli
-	kubectl kots remove ${app} -n gitpod --force --kubeconfig=${KUBECONFIG} || echo "No kots app existing, Installing"
-	$(MAKE) destroy-gitpod
-	export KUBECONFIG=${KUBECONFIG} && \
+	@echo "Cleaning up and re-installing Gitpod"
+	@kubectl kots remove ${app} -n gitpod --force --kubeconfig=${KUBECONFIG} || echo "No kots app existing, Installing"
+	@kubectl get secret -n gitpod https-certificates -o yaml > https-secret.yaml || echo "No existing certificate"
```

Comment:
**issue(bug):** I'm pretty sure that this will never backup the gitpod certificates; we delete the gitpod app _before_ attempting to backup the cert. Is this intended?

Reply:
Deleting the gitpod app doesn't cleanup the namespace. the secret will still be there.

Reply:
please see this PR  that fixes this issue out of the scope of this https://github.com/gitpod-io/gitpod/pull/13533